src/test/jtx/impl/Env.cpp

```diff
@@ -124,11 +125,17 @@ Env::close(NetClock::time_point closeTime,
         app().getOPs().acceptLedger(consensusDelay);
     else
     {
-        rpc("ledger_accept");
+        auto resp = rpc("ledger_accept");
+        if (resp["result"]["status"] != std::string("success"))
+        {
+            std::cerr << "Env::close() failed: " << resp["result"]["status"] << std::endl;
+            res = false;
+        }
         // VFALCO No error check?
```

Comment:
Looks like this comment can be removed now

Reply:
Done